components/raftstore/src/store/fsm/apply.rs

```diff
@@ -815,13 +817,16 @@ where
 
     /// The local metrics, and it will be flushed periodically.
     metrics: ApplyMetrics,
+
+    _phantom: PhantomData<ER>,
```

Comment:
ApplyDelegate does not need 'ER'. 'ER' could be only one type parameter of method which include `ApplyContext` as param.